app/Helpers/TerrafundDashboardQueryHelper.php

```diff
@@ -37,26 +36,36 @@ public static function getPolygonIdsOfProject($request)
         return $polygonsIds;
     }
 
-    public static function getPolygonsByStatusOfProject($request)
+    public static function getPolygonsByStatuses($projectUuid)
     {
-        $projectUuId = TerrafundDashboardQueryHelper::buildQueryFromRequest($request)
-        ->pluck('uuid')->first();
-        $project = Project::where('uuid', $projectUuId)->first();
+        $project = Project::where('uuid', $projectUuid)->first();
         $sitePolygons = $project->sitePolygons;
-
         $statuses = ['needs-more-info', 'submitted', 'approved'];
-
         $polygons = [];
 
         foreach ($statuses as $status) {
-            // Get polygons of the project filtered by status
             $polygonsOfProject = $sitePolygons
-            ->where('status', $status)
-            ->pluck('poly_id');
+                ->where('status', $status)
+                ->pluck('poly_id');
 
             $polygons[$status] = $polygonsOfProject;
         }
 
         return $polygons;
     }
+
+    public static function getPolygonsByStatusOfProject($request)
```

Comment:
This also has a very strange name. I suggest renaming it to `getPolygonsByProjectStatus`